tox.ini

```diff
@@ -11,24 +11,32 @@ deps =
   mypy: mypy~=0.711
 
 setenv =
-  PYTHONPATH={toxinidir}/opentelemetry-api/src/
-  mypy: MYPYPATH={env:PYTHONPATH}
+  mypy: MYPYPATH={toxinidir}/opentelemetry-api/src/
```

Comment:
Setting `PYTHONPATH` instead of installing the package was a kludge. It may be possible to get mypy to pass without setting `MYPYPATH` here, but just installing the packages didn't seem to do it.